src/amo/components/CollectionList/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import "~core/css/inc/mixins";
+@import "~photon-colors/photon-colors";
+@import "~ui/css/vars";
+
+.CollectionList {
+  padding: $padding-page;
+
+  @include respond-to(large) {
+    padding: $padding-page-l;
+  }
+
+  .Card-contents {
+    font-size: $font-size-m-smaller;
+  }
+}
+
+@include respond-to(large) {
+  .CollectionList-wrapper {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 0 $padding-page-l;
+    grid-template-columns: minmax(300px, 35%) 1fr;
+    margin: 10px 0 0;
+
+    .CollectionList-list.CardList {
+      margin: 0;
+    }
+  }
+}
+
+.CollectionList-create {
+  margin-top: 18px;
+  width: 100%;
+}
+
+.CollectionList-listing {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 0;
+  padding: 0;
+
+  .CollectionList-collection {
+    list-style-type: none;
+  }
+
+  a:link {
+    text-decoration: none;
+
```

Comment:
It would be nice to have some sort of UI feedback when we hover a collection (I mean more than the cursor). Maybe something like the `AddonsCard`?

![2018-06-25 14 31 41](https://user-images.githubusercontent.com/217628/41850360-828ad090-7884-11e8-8848-334c47cb1094.gif)